services/QuillLMS/app/models/auth_credential.rb

```diff
@@ -26,46 +27,27 @@
 #
 class AuthCredential < ApplicationRecord
   belongs_to :user
```

Comment:
I was thinking about the `has_one :canvas_auth_credential`, `has_one :google_auth_credential`, etc. but felt that would actually change how we use auth_credentials sitewide since it would allow for the possibility of multiple auth_credentials.  We should discuss offline

Reply:
I was actually thinking about the `has_one :canvas_auth_credential`, `has_one :google_auth_credential`, etc. yesterday, but felt that would actually change how we use auth_credentials sitewide since it would allow for the possibility of multiple auth_credentials.  We should probably discuss offline

Reply:
After discussing offline, Dan and I agreed that this is a good potential solution long term, but out of scope for this project.